sdk/core/core/test/curl_easy_perform_poc.c

```diff
@@ -47,12 +47,15 @@ int main() {
   }
 
   // Add auth
-  az_client_secret_credential credential = { 0 };
+  az_client_secret_credential credentials = { 0 };
```

Comment:
Can we parameterize Key Vault URL (i.e. antk-keyvault.vault.azure.net) to read it from the environment as well?

Reply:
And can you please also add "const" to "static az_span KEY_VAULT_URL = ..." at the top of this file (i.e. so that it becomes "static az_span const KEY_VAULT_URL")? The same for the other variables that don't change - API_VERSION_QUERY_NAME, API_VERSION_QUERY_VALUE, if you still need them after checking in the key vault client.